### PR TITLE
chore: upgrade @looker/components

### PR DESCRIPTION
**Pull request by noahamac**
upgrades @looker/components and cleans a few associated styles up. Tried to keep it focused

Mainly touches icons. Also replaces Spinner with ProgressCircular now that it is available

### CODE REVIEW

File: src/components/DiagramFrame/DiagramCanvas/DiagramToolbar.tsx [hunk 1]
```diff
@@ -32,6 +32,9 @@ import {
   IconButton,
   theme
 } from "@looker/components"
+import { CenterFocusWeak, Map } from "@styled-icons/material"
+import { Plus, Minus } from "@styled-icons/boxicons-regular"
```
Comment (bryans99) [inline src/components/DiagramFrame/DiagramCanvas/DiagramToolbar.tsx:36]: https://docs.google.com/spreadsheets/d/1J6S6a9cxH0O9wVaDtsEKXbusOf2vhFDFMU0uaK70U9c/edit#gid=0

I would expect Minus to be Remove from material
I would expect Plus to be Add from material

File: src/components/DiagramFrame/DiagramFrame.tsx [hunk 2]
```diff
@@ -30,6 +30,8 @@ import {
   IconButton,
   Layout,
 } from "@looker/components"
+import { Dashboard } from "@styled-icons/boxicons-solid"
```
Comment (bryans99) [inline src/components/DiagramFrame/DiagramFrame.tsx:33]: expect Dashboard from material

File: src/components/DiagramFrame/DiagramFrame.tsx [hunk 3]
```diff
@@ -138,7 +140,7 @@ export const DiagramFrame: React.FC<DiagramFrameProps> = ({
               borderRadius: "10px"}}
           />
           <IconButton
-            icon="GearOutline"
+            icon={<AccountTree />}
```
Comment (bryans99) [inline src/components/DiagramFrame/DiagramFrame.tsx:143]: Expect to be Settings from material-outline (I think thats the package, check with Jared)

File: src/components/DiagramFrame/DiagramFrame.tsx [hunk 4]
```diff
@@ -149,7 +151,7 @@ export const DiagramFrame: React.FC<DiagramFrameProps> = ({
               borderRadius: "10px"}}
           />
           <IconButton
-            icon="Tune"
+            icon={<Visibility />}
```
Comment (bryans99) [inline src/components/DiagramFrame/DiagramFrame.tsx:154]: Expect to be tune from material-outline

File: src/components/DiagramFrame/DiagramHeader.tsx [hunk 5]
```diff
@@ -53,14 +56,14 @@ export const DiagramHeader: React.FC<DiagramHeaderProps> = ({
         <Space gap="xsmall" justifyContent="flex-end">
           <IconButton 
             label="Explore Info" 
-            icon="CircleInfoOutline" 
+            icon={<InfoCircle />}
```
Comment (bryans99) [inline src/components/DiagramFrame/DiagramHeader.tsx:59]: Expect to be Info from material outlined

File: src/components/DiagramFrame/DiagramHeader.tsx [hunk 6]
```diff
@@ -53,14 +56,14 @@ export const DiagramHeader: React.FC<DiagramHeaderProps> = ({
...
             onClick={toggleExploreInfo}
             style={{color: selectionInfo.lookmlElement === "explore" && OVERRIDE_KEY, 
               backgroundColor: selectionInfo.lookmlElement === "explore" && OVERRIDE_KEY_SUBTLE,
               borderRadius: "25px"}}
             size="large" 
           />
-          <IconButton label="Reload Diagram" icon="Refresh" size="large" onClick={() => location.reload()} />
+          <IconButton label="Reload Diagram" icon={<RefreshOutline />} size="large" onClick={() => location.reload()} />
```
Comment (bryans99) [inline src/components/DiagramFrame/DiagramHeader.tsx:66]: Expect to be Refresh from material-outline

File: src/components/DiagramFrame/FramePanels/HelpPanel.tsx [hunk 7]
```diff
@@ -45,7 +45,7 @@ export const HelpPanel: React.FC<{
 }> = ({ 
  }) => {
   return (
-    <SettingsPanel width={`375px`} px="medium" py="large">
+    <SettingsPanel width={`275px`} px="medium" py="large">
```
Comment (bryans99) [inline src/components/DiagramFrame/FramePanels/HelpPanel.tsx:48]: width="275px"?

File: src/components/DiagramFrame/FramePanels/SettingsPanel/DiagramSettings.tsx [hunk 8]
```diff
@@ -87,8 +87,8 @@ export const DiagramSettings: React.FC<DiagramSettingsProps> = ({
               onChange={(value)=>{setSelectedBranch(value)}}
               disabled={(gitBranch && gitBranch.is_production) || !diagramExplore}
             />
-            <Divider appearance="light" my="medium" />
-            <Label fontSize="small" style={{marginTop: "0rem"}}>Select an Explore</Label>
+            <Divider appearance="light" my="medium" style={{height: 2}} />
```
Comment (bryans99) [inline src/components/DiagramFrame/FramePanels/SettingsPanel/DiagramSettings.tsx:90]: Hate these inline styles!

File: src/components/DiagramFrame/FramePanels/ViewOptions.tsx [hunk 9]
```diff
@@ -67,20 +71,20 @@ export const ViewOptions: React.FC<ViewOptionsProps> = ({
               onChange={setDisplayFieldType}
               options={[{label: "All fields", value: "all"}, {label: "Fields with joins", value: "joined"}]} />
           </FlexItem>
-          <Divider appearance="light" />
-          <FlexItem py="xsmall">
+          <Divider appearance="light" style={{height: 2}} />
```
Comment (bryans99) [inline src/components/DiagramFrame/FramePanels/ViewOptions.tsx:74]: This is the second time. Create a another component that exports `styled(Divider)`height: 2px;`

File: src/components/DiagramFrame/FramePanels/ViewOptions.tsx [hunk 10]
```diff
@@ -67,20 +71,20 @@ export const ViewOptions: React.FC<ViewOptionsProps> = ({
...
+          <FlexItem py="small">
             <Flex>
               <FlexItem>
                 <FieldToggleSwitch onChange={handleHiddenToggle} on={hiddenToggle} label="Hide hidden fields    " />
               </FlexItem>
               <FlexItem ml="xxxlarge">
                 <Tooltip content="Enabled by default, this toggle hides fields from the diagram that contain 'hidden: yes'.">
-                  <Icon size="xsmall" color="subdued" name="CircleInfoOutline"/>
+                  <Icon size="xsmall" color="subdued" icon={<InfoCircle />} />
```
Comment (bryans99) [inline src/components/DiagramFrame/FramePanels/ViewOptions.tsx:82]: Expect Info from material outlined

File: src/components/DiagramFrame/FramePanels/ViewOptions.tsx [hunk 11]
```diff
@@ -67,20 +71,20 @@ export const ViewOptions: React.FC<ViewOptionsProps> = ({
...
                 </Tooltip>
               </FlexItem>
             </Flex>
           </FlexItem>
-          <Divider appearance="light" />
+          <Divider appearance="light" style={{height: 2}} />
```
Comment (bryans99) [inline src/components/DiagramFrame/FramePanels/ViewOptions.tsx:87]: And again

File: src/components/DiagramFrame/FramePanels/ViewOptions.tsx [hunk 12]
```diff
@@ -138,7 +142,7 @@ export const ViewOptions: React.FC<ViewOptionsProps> = ({
                           <FlexItem>
                             <Icon 
                               size="xxsmall"
-                              name={viewVisible[item] ? "VisibilityOutline" : "VisibilityOff"}
+                              icon={viewVisible[item] ? <Visibility /> : <VisibilityOff />}
```
Comment (bryans99) [inline src/components/DiagramFrame/FramePanels/ViewOptions.tsx:145]: Visibility should be from material outline
VisibilityOff is correct

File: src/components/DiagramFrame/MetadataPanel/MetadataPanel.tsx [hunk 13]
```diff
@@ -126,8 +130,8 @@ export const MetadataPanel: React.FC<{
         }
         {metadata.primaryKey && 
           <Badge intent="neutral" size="small">
-            <Space gap="xsmall">
-              <Icon name="Key" size="xsmall" />
+            <Space gap="xxsmall">
+              <Icon icon={<PrimaryKey />} color={theme.colors.text3} pt="xxsmall"/>
```
Comment (bryans99) [inline src/components/DiagramFrame/MetadataPanel/MetadataPanel.tsx:134]: PrimaryLey should be VpnKey from material

File: src/components/DiagramFrame/MetadataPanel/MetadataPanel.tsx [hunk 14]
```diff
@@ -173,7 +179,7 @@ export const MetadataPanel: React.FC<{
           <ButtonTransparent
             mr="xxxlarge"
             ml="small"
-            iconBefore="LogoRings"
+            iconBefore={<LogoRings />}
```
Comment (bryans99) [inline src/components/DiagramFrame/MetadataPanel/MetadataPanel.tsx:182]: This is good!

File: src/components/DiagramFrame/MetadataPanel/MetadataPanel.tsx [hunk 15]
```diff
@@ -185,7 +191,7 @@ export const MetadataPanel: React.FC<{
           <ButtonTransparent
             ml="xxxlarge"
             mr="xsmall"
-            iconBefore="Explore"
+            iconBefore={<Explore />}
```
Comment (bryans99) [inline src/components/DiagramFrame/MetadataPanel/MetadataPanel.tsx:194]: And this

File: src/components/DiagramFrame/MetadataPanel/PrimaryKey.tsx [hunk 16]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```
Comment (bryans99) [inline src/components/DiagramFrame/MetadataPanel/PrimaryKey.tsx:1]: Dump this I think. See earlier comment

File: src/components/DiagramFrame/MetadataPanel/QueryChartButton.tsx [hunk 17]
```diff
@@ -43,7 +44,7 @@ export const QueryChartButton: React.FC<{
   return (
     <FlexItem>
       <ButtonOutline
-        iconBefore="CacheRefresh"
+        iconBefore={<Cached />}
```
Comment (bryans99) [inline src/components/DiagramFrame/MetadataPanel/QueryChartButton.tsx:47]: This is okay

File: src/components/DiagramFrame/utils.tsx [hunk 18]
```diff
@@ -1,5 +1,8 @@
+import React from "react"
+import { theme } from "@looker/components"
 import { IGitBranch, ILookmlModel, ILookmlModelExplore } from "@looker/sdk/lib/sdk/3.1/models"
 import { SelectOptionProps } from "@looker/components"
+import { GitBranch } from "@styled-icons/ionicons-solid"
```
Comment (bryans99) [inline src/components/DiagramFrame/utils.tsx:5]: I think this should come from looker icons

**Review by bryans99 — CHANGES_REQUESTED** [review]
Some of your icons are wrong I think.

See https://docs.google.com/spreadsheets/d/1J6S6a9cxH0O9wVaDtsEKXbusOf2vhFDFMU0uaK70U9c/edit#gid=0

File: src/components/DiagramFrame/DiagramFrame.tsx (hunk 2 above)
Comment (noahamac) [reply src/components/DiagramFrame/DiagramFrame.tsx:33]: This one will be removed in follow up PR

File: src/components/DiagramFrame/DiagramFrame.tsx (hunk 3 above)
Comment (noahamac) [reply src/components/DiagramFrame/DiagramFrame.tsx:143]: We using a new icon 
image

File: src/components/DiagramFrame/DiagramFrame.tsx (hunk 4 above)
Comment (noahamac) [reply src/components/DiagramFrame/DiagramFrame.tsx:154]: New icon 
image

File: package.json [hunk 19]
```diff
@@ -11,12 +11,18 @@
...
-    "@looker/components-test-utils": "^0.9.17",
-    "@looker/design-tokens": "^0.7.2",
+    "@looker/components": "^1.1.3",
+    "@looker/components-test-utils": "^1.1.3",
+    "@looker/design-tokens": "^1.1.3",
     "@looker/extension-sdk": "^0.11.0",
     "@looker/extension-sdk-react": "^0.6.2",
+    "@looker/icons": "1.1.3",
     "@looker/sdk": "^0.3.0-beta.1",
+    "@styled-icons/boxicons-regular": "^10.23.0",
```
Comment (bryans99) [inline package.json:21]: Get rid of non material and looker icons?

**Review by bryans99 — CHANGES_REQUESTED** [review]
Git rid of the non material and looker packages and we good to go

File: package.json (hunk 19 above)
Comment (noahamac) [reply package.json]: Oops. I always neglect the package.json

**Review by bryans99 — APPROVED** [review]
LGTM

It's possible we may need to explicitly reference material packages in the future when we turn on eslint. For the moment this is okay

**Pull request merged**